app/default/props.conf

```diff
@@ -10,12 +10,46 @@ TIME_FORMAT = %Y-%m-%dT%T.%6N%:z
 # Strip the RFC5424 header from the _raw field.
 SEDCMD-rfc5424_header = s/^\d+\s<\d{1,3}>\d\s(.+)\shost\s/\1 /
 
-TRANSFORMS-null = heroku_requests_doubleunderscore_null
+TRANSFORMS-source_type = heroku_system_source_type, heroku_app_source_type, heroku_requests_source_type, heroku_api_source_type, heroku_app_json_source_type
```

Comment:
The Splunk documentation suggests this shouldn't work in Splunk Cloud, but I've been told by Lars that it should. Other Splunk apps make use of this pattern.